src/main/java/net/sf/jabref/gui/help/HelpFiles.java

```diff
@@ -1,25 +1,141 @@
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
 package net.sf.jabref.gui.help;
 
-public class HelpFiles {
-    public static final String helpContents = "";
-    public static final String entryEditorHelp = "EntryEditorHelp";
-    public static final String stringEditorHelp = "StringEditorHelp";
-    public static final String searchHelp = "SearchHelp";
-    public static final String groupsHelp = "GroupsHelp";
-    public static final String contentSelectorHelp = "ContentSelectorHelp";
-    public static final String specialFieldsHelp = "SpecialFieldsHelp";
-    public static final String labelPatternHelp = "LabelPatterns";
-    public static final String ownerHelp = "OwnerHelp";
-    public static final String timeStampHelp = "TimeStampHelp";
-    public static final String exportCustomizationHelp = "CustomExports";
-    public static final String importCustomizationHelp = "CustomImports";
-    public static final String medlineHelp = "MedlineHelp";
-    public static final String generalFieldsHelp = "GeneralFields";
-    public static final String importInspectionHelp = "ImportInspectionDialog";
-    public static final String remoteHelp = "RemoteHelp";
-    public static final String journalAbbrHelp = "JournalAbbreviations";
-    public static final String regularExpressionSearchHelp = "ExternalFiles#RegularExpressionSearch";
-    public static final String nameFormatterHelp = "CustomExports#NameFormatter";
-    public static final String previewHelp = "PreviewHelp";
-    public static final String autosaveHelp = "Autosave";
+/**
+ * This enum globally defines all help pages with the name of the markdown file in the help repository at github
+ * @see <a href=https://github.com/JabRef/help.jabref.org>help.jabref.org@github</a>
+ *
+ *
+ */
+public enum HelpFiles {
+    COMMAND_LINE(
+            ""),
+    CONTENTS(
+            ""),
+    ENTRY_EDITOR(
+            "EntryEditorHelp"),
+    STRING_EDITOR(
+            "StringEditorHelp"),
+    SEARCH(
+            "SearchHelp"),
+    GROUP(
+            "GroupsHelp"),
+    CONTENT_SELECTOR(
+            "ContentSelectorHelp"),
+    SPECIAL_FIELDS(
+            "specialFieldsHelp"),
+    LABEL_PATTERN(
+            "labelPatternHelp"),
+    OWNER(
+            "OwnerHelp"),
+    TIMESTAMP(
+            "TimeStampHelp"),
+    CUSTOM_EXPORTS(
+            "CustomExports"),
+    CUSTOM_EXPORTS_NAME_FORMATTER(
+            "CustomExports#NameFormatter"),
+    CUSTOM_IMPORTS(
+            "CustomImports"),
+    GENERAL_FIELDS(
+            "GeneralFields"),
+    IMPORT_INSPECTION(
+            "ImportInspectionDialog"),
+    REMOTE(
+            "RemoteHelp"),
+    JOURNAL_ABBREV(
+            "JournalAbbreviations"),
+    REGEX_SEARCH(
+            "ExternalFiles#RegularExpressionSearch"),
+    PREVIEW(
+            "PreviewHelp"),
+    AUTOSAVE(
+            "Autosave"),
+
+    OPENOFFICE_LIBREOFFICE(
+            ""),
+
```

Comment:
@Siedlerchr Below I found
>   private final JButton help = new
>   HelpAction(Localization.lang("OpenOffice/LibreOffice integration"),
>            HelpFiles.OPENOFFICE_LIBREOFFICE).getHelpButton();

So, I am wondering if it is ok the line above is empty (out of curiosity, since I am no Java programmer).